frontend/test/pytest/test_jax_dynamic_api.py

```diff
@@ -602,5 +602,310 @@ def loop(i, _):
     assert_array_and_dtype_equal(res_a, jnp.ones([9, 3]))
 
 
+def test_qnode_whileloop_1():
```

Comment:
Can you add a test with abstracted_axes (i.e., the dynamism comes from outside the qnode and goes through the control flow operations)?

Reply:
Done. See [this](https://github.com/PennyLaneAI/catalyst/blob/e2c4869c518cc803fd405755839778bc4054c7f6/frontend/test/pytest/test_jax_dynamic_api.py#L585) and others